wire-library/wire-reflector/src/jvmMain/kotlin/com/squareup/wire/reflector/SchemaReflector.kt

```diff
@@ -54,6 +62,57 @@ class SchemaReflector(
     )
   }
 
+  private fun allExtensionNumbersOfType(type: String): ServerReflectionResponse {
+    val wireType = schema.getType(type)
+    if (wireType == null || wireType !is MessageType) {
```

Comment:
What's wrong with Enums?

Reply:
You can't have extensions for enum fields it seems.
I tried protoc compiling:
```
enum ColorType {
  RED = 0;
  BLUE = 1;
  GREEN = 2;
  extensions 100 to 199;
}
```
and it failed with `Missing numeric value for enum constant.`

However, the following works as expected:
```
message PublicNotification {
  required string content = 1;
  extensions 100 to 199;
}
```
(In proto2 only).